src/pages/selectGroup/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Button, Flex } from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+import Header from '@/common/components/header';
+import * as S from './index.styles';
+import { SelectNewGroupIcon, SelectRecentGroupIcon } from '@/assets/svgs';
+import { ROUTE } from '@/common/constants/route';
+
+function SelectGroup() {
+  const [selectedButton, setSelectedButton] = useState(0);
+
+  const navigate = useNavigate();
+
+  const handleButtonClick = (index: number) => {
+    if (selectedButton === index) {
+      return;
+    }
+    setSelectedButton((prev) => (prev === 0 ? 1 : 0));
+  };
+
+  return (
+    <>
+      {/** @Todo Header는 layout으로 분리 -> url 경로에 따라 나오게 변경 */}
+      <Header
+        title="모임 선택"
+        showIcon
+        type="TitleLeft"
+        handleBackButtonClick={() => console.log('뒤로가기 클릭')}
+      />
+      <Flex
+        direction="column"
+        justify="space-between"
+        mx="5"
+        height="100%"
+        mt="10px"
+        mb="32px"
+        flexGrow={1}
+      >
+        <main>
+          <Flex gap="2" direction="column" justify="flex-start" pb="120px">
+            <S.SmallContent>정산을 시작하려는</S.SmallContent>
+            <S.MainContent>모임을 선택해 주세요</S.MainContent>
```

Comment:
사소할수도 있는 부분이긴 한데. h1 태그가 페이지 안에 2개 있는 구조가 바람직한 구조인지 잘 모르겠습니다... 실제로 페이지 내에 2개 이상의 h1 태그가 있으면 크롤링 시에 오류가 발생하기도 하나봐요!! ([참고](https://galam.tistory.com/entry/%ED%8B%B0%EC%8A%A4%ED%86%A0%EB%A6%AC-%EB%B6%81%ED%81%B4%EB%9F%BD-%EC%8A%A4%ED%82%A8-SEO-%EA%B2%80%EC%83%89-%EC%B5%9C%EC%A0%81%ED%99%94-%EB%B0%A9%EB%B2%95-h1-%ED%83%9C%EA%B7%B8%EA%B0%80-2%EA%B0%9C%EC%9E%85%EB%8B%88%EB%8B%A4-%EB%AC%B8%EC%A0%9C-%EC%9B%90%EC%9D%B8%EA%B3%BC-%ED%95%B4%EA%B2%B0-%EB%B0%A9%EB%B2%95))
텍스트 스타일링 때문이었다면 아래처럼 `h1` 태그 안에서 `span` 태그를 이용해서 스타일링 하는 방법도 있었을 것 같습니당
(근데 계속 페이지를 보다 보니 헤더의 "모임 선택" 이라는 문구가 좀 더 h1에 어울리는 것 같기도 하네요... 😵‍💫)

```typescript
export const SmallContent = styled.span`
  display: block;
  font-size: 16px;
  font-weight: 500;
`;

export const MainContent = styled.span`
  display: block;
  font-size: 24px;
  font-weight: 600;
`;
```

```suggestion
            <h1>
              <S.SmallContent>정산을 시작하려는</S.SmallContent>
              <S.MainContent>모임을 선택해 주세요</S.MainContent>
            </h1>
```

Reply:
아 h1 태그는 페이지 당 하나만 쓰는게 맞네요! header 컴포넌트의 제목은 \<h2> 태그를 주로 쓰는 것 같습니당  
\[구획 컨텐츠 테이블 - MDN\](https://developer.mozilla.org/ko/docs/Web/HTML/Element/Heading_Elements#%EA%B5%AC%ED%9A%8D_%EC%BD%98%ED%85%90%EC%B8%A0_%EB%A0%88%EC%9D%B4%EB%B8%94)

```javascript
<header>
  <nav aria-labelledby="primary-navigation">
    <h2 id="primary-navigation">주 탐색창</h2>
    <!-- navigation items -->
  </nav>
</header>
```